charts/akeyless-secure-remote-access/values.yaml

```diff
@@ -21,6 +21,9 @@ legacySigningAlg: "false"
 # Optional, provide a key-name to extract the username (value) of that key in sub-claims. The extracted username will be used to authenticated to the remote target (RDP or SSH).
 usernameSubClaim:
 
+# Comma separated list of the URLs that will be considered valid for redirection to this bastion (security measure to prevent Open Redirect)
+allowedBastionUrls:
+
```

Comment:
it might better to put here a comment with an example to the expected input

Reply:
added example